test/http_mock_adapter_test.dart

```diff
@@ -63,6 +67,64 @@ void main() {
           () => dioAdapter.onPatch(path).reply(statusCode, data));
     });
 
+    group('Dio adapter mock get', () {
```

Comment:
Could you please add or change code in a way that also tests _DioAdapter_-s functionality such as `onRoute()` function and its wrappers (`onGet()`, `onPost()`, etc.)?

Reply:
კი,მივამატებ მაგას.

Reply:
Slack შეამოწმე როცა შეძლებ.